### DISCUSSION TITLE
I seem to have chosen poorly: what on earth is going on?

### DISCUSSION THREAD

**Discussion by airwindows** [latenitefilms/BRAWToolbox#118] [Support] [answered]
I bought BRAW Toolbox to edit stuff in Final Cut, having done a video shoot with my BCC4K still plugged into an ATEM. It forced the camera into BRAW, and I thought I would try working with that.

Though it's a bit cumbersome, I greatly enjoyed what seems to me to be a big video quality boost through doing my color correcting mostly with the BRAW controls such as color temperature, contrast and tint rather than stuff like color boards and curves. I got pretty excited and spent some hours finessing this.

Lining stuff up in the timeline, I layered multiple camera feeds and a screen capture and prepared to do my usual thing of trimming the front off the footage, and then repeatedly cmd-shift-B 'blade all' to separate sections of video and alter opacity or audio volume of different layers (or delete stuff that's totally unused.

With the very first cut, I was ruined. BRAWs invariably switch to the beginning of the source footage (for video, not audio?) at every edit, every adjustment. I couldn't even trim the front off the clip and go from there by editing the surrounding clips. I am downloading Resolve, in despair, to be able to even get my work done for the late Sunday posting I need to do. I couldn't find any way to export out a transcode or proxy file.

Are the BRAW files in the timeline meant to be editable in any way? The backbone of my whole video is constructed out of the one BRAW file the camera stuck me with. My workflow doesn't involve grabbing individual clips out of the library as I have multiple audio/video tracks that all must be synchronized at a clap/cue noise at the beginning of the file, and edited from there.

Working with the BRAW masters and tailoring them within BRAW to my visual needs was so promising. What was I supposed to be doing? Maybe if there is any possible way for me to work with this, I can have a better game plan by next Sunday and another video. I'm kicking myself for discarding my hours of work now, as I could have just exported the underlying video, looking the way I wanted, as a ProRes export, and then it would have literally been the transcoding I'm downloading Resolve to be able to do…

…quick edit, in fact I DID make a preset out of my color adjustment and so I have begun exporting the ProRes underlying video, which will work as desired and function as my 'transcode' without a need to download Resolve. woot! I would still really like to know what I was supposed to do. I am puzzled by my inability to make timeline editing work with BRAW Toolbox.

**Comment by latenitefilms** [comment]
Are you on the latest Ventura by any chance? We've had a few reports now of Final Cut Pro doing bizarre things if you're working with Synchronised Clips.

The workaround seems to be to always use Multicam Clips - see:

https://brawtoolbox.io/faq/#i-dont-like-synchronised-clips-can-i-import-a-multicam-clips-instead

If you're already started editing, one potential workaround is to "Open Clip" on your existing Synchronised Clip and make the audio track and the black generator a Compound Clip - this seems to fix things.

I'm still on Monterey on my main machine - so I still need to do some more digging on a Ventura machine and chat with the Final Cut Pro team to work out exactly what's going wrong.

**Reply by airwindows** [reply]
Good to hear from you! I'm not on Ventura, I'm on Monterey 12.5.1. I looked up as much info as I could and had already tried making Multicam Clips and doing an edit: same thing, as soon as I used the Blade Tool every cut made every resulting clip start its video at the beginning of the parent clip. I did NOT try 'Open Clip' though I was searching for anything I could try. Is this Open Clip in Final Cut or is it back in BRAW Toolbox? If there's a way to get my expected behavior I'll be good: I commonly write stuff down and can follow complicated and arbitrary workflows if they'll get me there.

From saying 'audio track and the black generator' it sounds very much like you mean it's in Final Cut space, as Final Cut would be seeing an audio clip and a black generator which BRAW Toolbox would be replacing with its output inside the FX. Again, I tried a multicam clip and got the same issue, so I was trying to figure out if it was simply impossible to edit in the timeline at all (which would be real weird). I have a transcoded version I'm making today's video with, but I'll keep the problematic BRAW and once I've exported my video to upload today, I'll get back to square one and try and find out what's happening and whether 'Open Clip' does anything useful or not :) Again, I'm on Monterey, so I'd be surprised if it's a Ventura issue.

**Reply by latenitefilms** [reply]
Just to confirm... you're definitely using the Workflow Extension to create the BRAW Clips, right (as opposed to manually applying the BRAW Toolbox effect)?

See: https://brawtoolbox.io/installation/

You should never have to manually apply the BRAW Toolbox effect.

**Reply by airwindows** [reply]
Interesting. There's a button at the top left of the window, where I can drag in BRAW and choose an option to make a green button that produces stuff that I can only drag into a library, and that creates a clip that I can put into the timeline. I'd thought I needed to also manually apply the BRAW toolbox effect to alter anything.

I don't remember whether I've edited a clip that has NOT had anything done in this way. My unaltered BRAWs weren't acceptable and needed adjusting. I don't remember seeing where to do that, if not by applying the effect… and I don't know how you'd save presets to the effects menu without having applied an effect.

So what I've done is produce clips in the way I described above, and then I have manually applied the effect to get access to the controls inside it. Sounds like I need to dig deeper to work out how this is done, as I'm applying aspects of my workflow (like dragging custom effects to clips so it's always the same color correction, etc, and only then editing the clip into lots of little parts with changes) and that's where I'm choosing poorly: something about what I normally do isn't translating.

Pretty sure the only way to get to a BRAW clip that can be dragged into the timeline is with what you're calling the Workflow Extension, and that the problem is in me building a workflow out of manually dragging a preset that originally derives from manually applying the BRAW Toolbox effect, thinking that was how you edited it.

**Reply by latenitefilms** [reply]
Ok, so it sounds like you're correctly using the Workflow Extension to create BRAW Clips, but you're incorrectly manually adding the BRAW Toolbox effect to the created Synchronised Clips (or Multicam Clips) to attempt to access the RAW controls.

To access the RAW controls instead, you should double click on the Synchronised Clip (or select **Clip > Open Clip** from the Final Cut Pro main menu), which will then show a Black Generator and a Audio file like the below:

<img width="2672" alt="image" src="https://user-images.githubusercontent.com/22286696/233979315-e6b274ce-48fc-4b2c-8a69-771dd40c7428.png">

If you have the **Create Multicam Clips** option selected in the BRAW Toolbox Workflow Extension (which is recommended), then you need to go inside the Multicam Clip, THEN inside the Synchronised Clip on the first angle.

This is explained in the [Installation instructions](https://brawtoolbox.io/installation/), however I'll review this documentation again to see if we need to make it more clear, or add more screenshots.

Hope this clarifies things! Any questions let me know!

**Reply by airwindows** [reply]
> Hope this clarifies things! Any questions let me know!

Questions, I do indeed have :)

I can load in the BRAW as before. If I don't do anything, then I can edit them and it seems to work.

It doesn't look like I can go inside a clip on the timeline, to further alter settings. It looks quite a bit like I can prepare all the clips before dragging them to the library, and as long as I'm clear on exactly what I'll want to do, they will be editable as normal. I'll take it.

I've just gone into the multicam and found Clip>Open Clip and got to a place where, it appeared I was working on the raw source clip rather than the part inside the timeline and its edits. When I can work with that stuff, it reveals no internal BRAW settings to be edited.

When I go to where I can find that, I got the clip with some boundaries that weren't quite the selections or edits I'd made. I was able to alter color temperature, but then couldn't get out again. Re-opening the project got me a result where part of the video was the new color temp, and part was the old. Unfortunately the point of transition restarted the library clip from its beginning again, exactly the problem I had before.

I'm not sure I was ever meant to do on-the-fly alterations to the BRAW controls. Are they not just import settings? I'm way closer to a useable workflow, but only when I take pains to do all BRAW settings only on import, and never attempt to touch them once actively editing the project. Again, I'll take it, but I can't help but think of the things I could do if I could manipulate these things on individual edits made to the overall clip. That's where I'm at, on revisiting the situation :) better than it was! Curious if I'm still missing something.

Weirdly, if I have a 'broken' file that is restarting (and not synced video to audio) and I cut it, it remains desynched. But if I drag the new section over the top of the previous layer, I then get both parts playing, correctly synched, each with its own distinct color temperature in play. Then when I put it back in the same layer again, it returns to desynched.

Then, if I drag it onto the second layer and drag out the clip start until it's nearer the library clip start, it restores itself. Unexpected. Basically I'm finding that if I try to do anything with PARTS of a BRAW clip in the library, I can put in changes, but they're not associated with the timeline very well and they're being done to the source clip in the library, and not as part of the timeline itself. I'm finding this kind of unintuitive and balky, and will probably try to avoid using that functionality (or perhaps trying to pre-edit stuff either on import, or making edits on the clip itself and altering stuff, before attempting to drag it into the timeline)

tl:dr; I still can't alter BRAW parameters effectively in the timeline (to 'instances' of what is actually a clip in the library?) but I can do some of what I need by focussing on either import, or doing it to the clips before bringing them to the timeline. And in some cases when clips 'de-synch' they can be put back by further editing, dragging timeline clips around and resizing them :)

**Reply by latenitefilms** [reply, marked as answer]
You can definitely do on-the-fly RAW adjustments - you can even keyframe the ISO, Exposure, Color Temp, Tint and Custom Gamma Controls (which is something even DaVinci Resolve can't currently do).

In a Final Cut Pro timeline you can double click on a Synchronised Clip to open the "insides" of that Synchronised Clip in its own timeline. When you double click on a BRAW Toolbox Synchronised Clip, you'll see something like this, where you can manipulate the RAW parameters:

<img width="2168" alt="image" src="https://user-images.githubusercontent.com/22286696/234199197-623c27e1-ab0f-4795-a222-45c6ec94d7da.png">

However, Synchronised Clips are always their own instances. For example, if you have a Synchronised Clip in your timeline, if you modify the Synchronised Clip in your Browser, the changes won't ripple across - the Synchronised Clip instance in your browser is unique, and different to the instance in your timeline. 

Similarly, if you have a single Synchronised Clip in your timeline, and use the blade two to make it into two clips, now both those clips are their own unique instances. You can rename one clip, and it won't affect the other. You can double click one clip, add a Color Board effect and manipulate it, then when you jump back to your project, that Color effect will only be applied to one of those clips, not both.

For example, before:

<img width="336" alt="image" src="https://user-images.githubusercontent.com/22286696/234200438-e52be053-f741-4583-a308-5a9f9ec438c9.png">

...and after:

<img width="341" alt="image" src="https://user-images.githubusercontent.com/22286696/234200548-bd1c5435-ed5b-4427-9e51-da7135f738bd.png">

Generally speaking however, we always recommend using the **Create Multicam Clips** feature in the BRAW Toolbox Workflow Extension (explained [here](https://brawtoolbox.io/faq/#i-dont-like-synchronised-clips-can-i-import-a-multicam-clips-instead)). Multicam Clips work differently to Synchronised Clip's because any changes you make to a Multicam Clip in your Browser will flow through to all instances of that Multicam Clip in all your Projects/Timelines.

When you have **Create Multicam Clips** ticked, BRAW Toolbox will create a Multicam Clip for each BRAW file, and within that Multicam Clip, there's a single angle with a Synchronised Clip. For example:

<img width="1477" alt="image" src="https://user-images.githubusercontent.com/22286696/234200836-eb112230-0e29-4e26-9810-befab8807e21.png">

To modify the RAW properties, you again simply double click on the Synchronised Clip, and access the RAW properties via the Generator on the Primary Storyline:

<img width="2168" alt="image" src="https://user-images.githubusercontent.com/22286696/234200977-a67bcdea-486b-47c2-8958-d28cffa094c3.png">

Because the Synchronised Clip lives inside the Multicam Clip, there's only one instance of the Synchronised Clip - and therefore if you make any changes to it, it'll flow through to ALL instances of the Multicam Clip - i.e. if you change the saturation in the Synchronised Clip, all your projects/timelines that have that Multicam Clip will update.

Essentially what I recommend doing is:

1. In the Workflow Extension you should get all your RAW settings as close to what you want as possible. Consider this stage your "dailies", where you want to make the rushes look as good as possible.
2. Make sure you have **Create Multicam Clips** ticked in the Workflow Extension.
3. You can manipulate the RAW properties by double-clicking on a Multicam Clip in your Browser, then double-clicking on the Synchronised Clip, selecting the Black Solid on the primary storyline, then going to the Video Inspector to adjust the BRAW Toolbox effect. You can also apply other Colour Correction effects after the BRAW Toolbox effect if required too.
4. You can then edit with those Multicam Clips in your project/timeline. If you want to tweak the RAW parameters, simply double click on the Multicam Clip (regardless of whether it's in the Browser or Timeline), then double click the Synchronised Clip, and adjust as you did in the previous step.

I also recommend Granting Sandbox Access to whatever drive contains your BRAW footage, as there's a very slight performance benefit of doing this. You can learn more about this [here](https://brawtoolbox.io/collaboration/).

If you haven't already, it's worth having a watch of the Installation Video here:

https://brawtoolbox.io/installation/

I also explain this in more detail in this Ripple Training video here:

https://brawtoolbox.io/videos/

**Comment by airwindows** [comment]
I'm not finding a way to do changes cleanly to the Synchronised Clip: I do see that I can manipulate it, but it seems like I'd better try and edit it in Clip form, not in the project timeline. If there are special effects I have in mind I'm game to attempt them there: have already done changes, just didn't feel they were very controllable and I had trouble with them.

I'm having no trouble working with the resulting Multicam/Synchronized clips in the project timeline now that I know to alter the BRAW stuff elsewhere, and I'm very pleased with the quality jump, so on the whole I'm quite happy with minor alterations in workflow.

Thanks! I'm happy to have this discussion closed as completed :)

**Reply by latenitefilms** [reply]
Great news! Happy cutting!

We'll continue to work with Apple to try and solve the weird Synchronised Clip bug. I think Apple probably just never expected users to put just a Generator in a Synchronised Clip.
